### PR TITLE
Example messages

### PR DESCRIPTION
**Pull request by aaltat**
Does not work and now idea how do that with TS. I looked at Python and.node examples, but could not figure out a way to:
1) Read send parameters by one parameter at the time
1) Send response that contains data

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -2,19 +2,21 @@ import { IPlaywrightServer, PlaywrightService } from './generated/playwright_grp
...
         await this.browser.close();
         callback(null, new Empty());
     }
 
-    async openBrowser(call: ServerUnaryCall<Empty>, callback: sendUnaryData<Empty>): Promise<void> {
+    async openBrowser(call: ServerUnaryCall<KeywordRequest>, callback: sendUnaryData<Response>): Promise<void> {
+        console.log("Open browser: " + call.request)
         this.browser = await firefox.launch({headless:false});
-        callback(null, new Empty());
+        callback(null, {rc: 'foo'});
```
Comment (mkorpela) [inline src/index.ts:19]:
```
-        callback(null, {rc: 'foo'});
+        const response = new Response();
+        response.setRc('foo');
+        callback(null, response);
```

**Comment by mkorpela** [comment]
Seems to work after the change I made.
```
Listening on 50051
Open browser: firefox
Close browser: alias 1,foo
Open browser: chrome
Close browser: alias 1,foo
Open browser: firefox
Close browser: alias 1,foo
Open browser: firefox
Close browser: alias 1,foo
Open browser: firefox
Close browser: alias 1,foo
```

**Comment by aaltat** [comment]
Thanks for the pointers. Not ready to merge but somewhat more closer to ready.

**Comment by mkorpela** [comment]
I think the error handling part should be done with grpc error handling mechanism https://grpc.io/docs/guides/error/

**Comment by aaltat** [comment]
Even better with this https://cloud.google.com/apis/design/errors#error_model

What about logging?

**Pull request merged**